tempesta_fw/http.c

```diff
@@ -1153,28 +1173,32 @@ tfw_http_resp_cache_cb(TfwHttpReq *req, TfwHttpResp *resp)
  * Request messages that were forwarded to a backend server are added
  * to and kept in @msg_queue of the connection @conn for that server.
  * If a paired request is not found, then the response is deleted.
+ *
+ * If thre isn't paired client request, then it seems upsream server is
```

Comment:
It appears that the comment is garbled. Perhaps, "*there isn't* a paired client request"?
Or, perhaps: `If a paired client request is missing, then...`